modules/apps/portal-template/portal-template-freemarker/src/main/resources/FTL_liferay.ftl

```diff
@@ -66,9 +66,9 @@ LPS-30525.
 	file_name
 >
 	<#if file_name == css_main_file>
-		<link class="lfr-css-file" href="${file_name}" id="mainLiferayThemeCSS" rel="stylesheet" type="text/css" />
+		<link class="lfr-css-file" href="${htmlUtil.escape(portalUtil.getStaticResourceURL(request,file_name))}" id="mainLiferayThemeCSS" rel="stylesheet" type="text/css" />
```

Comment:
Question about formatting: do we expect a space after the `request,`?

Reply:
Yes, as you can see in line 23: 

`htmlUtil.escape(portalUtil.getStaticResourceURL(request, "${themeDisplay.getPathThemeCss()}/main.css"))` 

SF in freemarker is almost useless, so Brian usually fixes manually all of these.

Reply:
So, up to you @jgarcialr: you can either add another commit with the formatting fixes just to be safe, or go for the coin toss that Brian will do a "please resend" or a "see `1234abcd`" 😁

Reply:
Thanks for the tips :grin: 
I'll go with the safe bet.